src/api/todos.ts

```diff
@@ -1,8 +1,20 @@
 import { Todo } from '../types/Todo';
 import { client } from '../utils/fetchClient';
 
-export const getTodos = (userId: number) => {
+export const getTodos = (userId: number | undefined) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
+export const createTodos = (title: string) => {
+  return client.post<Todo[]>('/todos', {
```

Comment:
You don't need to implement creating todos in this part of the task. It'll be needed later. So remove it for now